### DISCUSSION TITLE
ALL renaming done maxGraph/maxGraph !

### DISCUSSION THREAD

**Discussion by coclav** [maxGraph/maxGraph#47] [Development]
maxGraph organisation / maxGraph project

Sorry for inconvenience of everyone who checked out the project already. Now we are off on solid foundation :-) 

Let's get to work on v5. I hope to have more time for it this summer.

**Comment by jgadsden** [comment]
@coclav do we want to preserve the main branch as the original version of mxGraph? Or create a new branch so that we preserve it as it was (for reference)

The release branch of maxGraph/maxGraph then can have some reverts done (if necessary) and then be released as a continuation of jgraph/mxgraph version 4.2.2 (probably 4.2.3)?

**Reply by coclav** [reply]
I'm not a versioning expert (hence my time to reply). 

I'd suggest:
*  ex-mxGraph 4.2.2 released under the maxGraph name version 4.2.3 
* add notification in 4.2.3 in npm to announce upcoming change (I thing we can do that right?)
* release the typescript version for 5.0.0 

does that make sense ? @mcyph @jgadsden 

@mcyph would you like to get in touch with me direct colin@getskore.com ?

**Comment by lalicw** [comment]
will this change the npm package name to maxgraph?

**Reply by tbouffard** [reply]
yes, we can not use the `mxgraph` name which is own by the former maintainers of the project. They also don't want us to reuse the mxgraph name for any resources related to our new project.

**Reply by coclav** [reply]
YEs this is the project

**Reply by lalicw** [reply]
ok, thanks for reply, hope for the new npm package
